Library/Formula/cv.rb

```diff
@@ -0,0 +1,12 @@
+require "formula"
+
+class Cv < Formula
+  homepage "https://github.com/BestPig/cv"
```

Comment:
Can’t you use [BestPig’s repo](https://github.com/BestPig/cv/releases/tag/v0.5.1) in the meantime? It’s linked in the upstream README as the official OSX port until they merge the support upstream.